modules/c++/nitf/source/ImageWriter.cpp

```diff
@@ -56,14 +53,19 @@ void ImageWriter::setWriteCaching(int enable)
 
 void ImageWriter::setDirectBlockWrite(int enable)
 {
-    if (!nitf_ImageWriter_setDirectBlockWrite(getNativeOrThrow(), enable, &error))
+    if (!nitf_ImageWriter_setDirectBlockWrite(getNativeOrThrow(),
+                                              enable,
+                                              &error))
     {
         throw nitf::NITFException(&error);
     }
 }
 
 void ImageWriter::setPadPixel(nitf::Uint8* value, nitf::Uint32 length)
 {
-    if (!nitf_ImageWriter_setPadPixel(getNativeOrThrow(), value, length, &error))
+    if (!nitf_ImageWriter_setPadPixel(
+                getNativeOrThrow(), value, length, &error))
```

Comment:
why the different style formatting between these two? (line 56 vs 55)

Reply:
I added an .clang-format auto-formatting script a while back. I'm gradually applying it to files as I touch them.